OpenTrons/opentrons-api
# fix(app): fix add missing banner for runtime parameters

# Overview
fix add missing banner for runtime parameters

close RQA-3015


## Test Plan and Hands on Testing
1. go to device detail page
2 go to run history
3. click view a run from overflow menu
4. check that the banner (title is download) is displayed


## Changelog
- add a banner for a terminated run (from overflow menu in historical runs)
- add a test for the above case



## Review requests



## Risk assessment
low
